plant-crossing/screens/GardenScreen.tsx

```diff
@@ -57,7 +59,8 @@ export default function GardenScreen() {
   return (
     <View style={styles.container}>
       <Text style={styles.title}>Garden Page</Text>
```

Comment:
do you think you could get rid of this line so that it does not say garden screen at the top anymore please